packages/react/src/useContextSelector.ts

```diff
@@ -53,8 +61,7 @@ function useContextSelector(context, selector, equalityFn = shallowEqual) {
   };
 
   // Update the selector fn to memoize the selected value by [equalityFn].
-  const patchedSelector = equalityFn ? equalityFnCallback : selector;
```

Comment:
**💡 thought:**

Hm, if it would be called with `equalityFn = null`, would it fail now?